clash-prelude/src/Clash/Signal/Delayed/Internal.hs

```diff
@@ -141,7 +141,7 @@ fromSignal = coerce
 
 -- | __EXPERIMENTAL__
 --
--- __Unsafely__ convert a 'Signal' to /any/ 'DSignal' clk'.
+-- __Unsafely__ convert a 'Signal' @dom@ to /any/ 'DSignal' @dom@.
```

Comment:
I initially read this as: ... convert ... to any domain.
I think it would be clearer to explicitly say the `any` applies to the delay, something like:
```suggestion
-- __Unsafely__ convert a 'Signal' to a 'DSignal' with an arbitrary @delay@.
```